skimage/filters/thresholding.py

```diff
@@ -727,14 +727,10 @@ def threshold_triangle(image, nbins=256):
     width /= norm
 
     # Maximize the length.
-    d = peak_height * arg_low_level - width * hist[arg_low_level]
```

Comment:
I'm all for additional comments there to explain the calculation -- I had to look at the reference to see the geometry -- but I'm still not sure I understand where that d came from. I got the equation that is now line 730 from comparing some similar triangles. Do you have a suggested comment that explains the original equation?